runners/core-java/src/main/java/org/apache/beam/runners/core/triggers/AfterPaneStateMachine.java

```diff
@@ -40,24 +40,24 @@
       StateTags.makeSystemTagInternal(StateTags.combiningValueFromInputInternal(
           "count", VarLongCoder.of(), Sum.ofLongs()));
 
-  private final int countElems;
+  private final long countElems;
 
-  private AfterPaneStateMachine(int countElems) {
+  private AfterPaneStateMachine(long countElems) {
```

Comment:
Changed back to int, and shrunk the value in the proto to make it clear. Note that we could check for overflow, but we actually want to do a blind write to the accumulator.